Makefile

```diff
@@ -26,6 +27,9 @@ endif # TARGET
 ASSISTED_ORG := $(or ${ASSISTED_ORG},quay.io/ocpmetal)
 ASSISTED_TAG := $(or ${ASSISTED_TAG},latest)
 
+# IMAGE_FORMAT set by openshift ci to <registry>/<namespace>/stable:${component}
+component = assisted-service
+export SERVICE := $(IMAGE_FORMAT)
```

Comment:
Why? You can set directly `SERVICE` by OpenShift CI

Reply:
Can you elaborate on that? What did you have in mind?

Reply:
I am not sure where this `IMAGE_FORMAT` var is coming from. But instead of creating it, and then assign it to `SERVICE`...
Why not setting directly `SERVICE` to be the `IMAGE_FORMAT` value? If I understand correctly it's coming from the OpenShift CI configuration file

Reply:
`IMAGE_FORMAT` is indeed defined (out of the box) by openshift ci. It has the format mentioned in the comments, in which `registry` and `namespace` are different on each build (mainly `namespace` is different - `registry` could be different).
The `component` part at the end is literally the string `${component}`. The approach we took then is to define the value of component here, and then apply value to `SERVICE` by expanding `IMAGE_FORMAT` once `component` is defined.
Then `SERVICE` can be used in the same way it was used in the makefile up until now.

Reply:
I wasn't familiar with that env var. It isn't depicted on https://docs.ci.openshift.org/docs/architecture/step-registry/#available-environment-variables
Can `SERVICE` be defined on OpenShift CI? by the `env` stanza

Reply:
I don't think it's possible, but let me try and verify that.

Reply:
ok. Not sure if the `env` stanza is a possibility, but one thing I can suggest and I think should work is doing something like this when calling the deployment target:

```
SERVICE=$(echo "$IMAGE_FORMAT" | component=assisted-service envsubst) make deploy-on-openshift-ci
```

As the component name is actually set in the config yaml file (`to: assisted-service` under `images:`), it could actually make sense to have `component=assisted-service` on the the same yaml file rather than on the makefile.

Let us know if this makes sense to you?